src/HttpHeader.cc

```diff
@@ -1469,7 +1487,7 @@ HttpHeaderEntry::parse(const char *field_start, const char *field_end)
 
     if (field_end - value_start > 65534) {
         /* String must be LESS THAN 64K and it adds a terminating NULL */
-        debugs(55, DBG_IMPORTANT, "WARNING: ignoring '" << theName << "' header of " << (field_end - value_start) << " bytes");
+        debugs(55, DBG_IMPORTANT, "WARNING: rejecting due to header of " << (field_end - value_start) << " bytes");
```

Comment:
I would still report the header name here because many admins may find it useful (especially the few admins that are going to find this message itself useful). If needed, use the Raw interface for reporting the name. I do not insist on restoring name reporting.

Reply:
Okay. Done